targets/cortex-m-qemu.json

```diff
@@ -2,6 +2,7 @@
 	"inherits": ["cortex-m3"],
 	"build-tags": ["qemu", "lm3s6965"],
 	"linkerscript": "targets/lm3s6965.ld",
+        "default-stack-size": 4096,
```

Comment:
Nit: please use tabs here for consistency.

Reply:
I'll fix this in a later PR so we can merge this one.